app/ember/package.json

```diff
@@ -28,12 +28,15 @@
     "@babel/runtime": "^7.0.0",
     "@ember/test-helpers": "^0.7.25",
     "@storybook/core": "4.0.0-alpha.24",
+    "common-tags": "^1.8.0",
+    "moment": "^2.22.2",
```

Comment:
I think `moment` is not needed here. It comes from knobs AFAIR.

Reply:
will remove, it was being required when I tried to install